### PR TITLE
Improve discussion of delimiters

### LINKED ISSUE #76: Null values should be easily distinguishable from the delimiter
From JE:

Matt Davis:
Under “Use good null values” I would add that you should pick a null value that is easily distinguishable from the table delimiter. I have seen tab separated tables where the null value was a space…

### LINKED ISSUE #80: Address the use of comma delimiters with European decimal notation
From JE:

Tom Webb:

> One minor issue that you may want to consider concerns your recommendation of .csv as a good format for text files. I agree – and it’s the format I use all the time in my own work. However, I also collaborate with people in continental Europe, where commas are used as the decimal separator (i.e., 1.2 is represented as 1,2). This has caused problems before for me, especially when people are converting Excel tables into text files which I then read into something like R. It is easily addressed if you know it’s an issue, but if you’re unaware of it, it can cause headaches! So maybe just adding a sentence somewhere which says that you should state your decimal separator (at least, if it’s not ‘.’) may be useful.

Beroe:

> Agree about commas = troublesome. CSV has come to mean generic delimited files rather than true comma-separated values in a lot of contexts, so recommend TABs.

Ethan:

> I’m not convinced at this point that Tabs are superior delimiters. Tabs have their own issues: 1) they can display differently in different editors meaning that data that is readable by the person creating the data may be very difficult to view in the raw form by someone else; 2) It can be difficult to tell the difference between tab delimited data, space delimited data, and fixed width data; 3) most importers default to commas as the delimiter. You can get around all of these of course, but it’s its own set of hurdles, particularly for less experienced analysts. We’ll definitely caution against using commas with European style decimals, but I’d need more convincing that tabs are inherently superior.

### PR DESCRIPTION
**Pull request by ethanwhite**
1. Address issue of comma delimiters in continental Europe. Fixes #80.
2. Address issue of tab/space delimiters with blanks for nulls. Fixes #76.

### CODE REVIEW

**Comment by dmcglinn** [comment]
Looks good just one minor typo:
change "There are two potential issue ..." to "There are two potential issues ..."

**Comment by ethanwhite** [comment]
Thanks. Fixed.

File: data_sharing_ms.md [hunk 1]
```diff
@@ -180,7 +181,10 @@ The null value that is most compatible with the software commonly used by biolog
 Blanks are automatically treated as null values by R, Python, SQL, and Excel.
 They are also easily spotted in a visual examination of the data.
 Note that a blank involves entering nothing, it is not a space, so if you use this option make sure there aren't any hidden spaces.
-One potential issue with blanks is that it can be difficult to know if a value is missing or was overlooked during data entry.
+There are two potential issues with blanks that should be considered:
+
+1. It can be difficult to know if a value is missing or was overlooked during data entry.
+2. They can be confusing when spaces or tabs are used as delimitors in text files.
```
Comment (embaldridge) [inline data_sharing_ms.md]: delimiters

**Comment by ethanwhite** [comment]
Fixed the 'delimitors' typo and removed the extra correction of the data plural issue that was preventing an automated merge.

**Pull request merged**